### DISCUSSION TITLE
Supporting gRPC/Protobuf with Buf!

### DISCUSSION THREAD

**Discussion by perezd** [chapar-rest/chapar#29] [Ideas]
I saw your Reddit post mentioning interest in gRPC/Protobuf support. Over at Buf, we're gophers as well, and we have a lot of tooling that could be helpful to bring gRPC support without a ton of heavy lifting.  Check out: https://github.com/connectrpc/connect-go

It's a fully compatible gRPC library for clients and servers that you could easily integrate to support gRPC.

We also offer a straightforward reflection API (powered by our schema registry). 
It was recently integrated into Insomnia, which is a very similar tool:
https://docs.insomnia.rest/insomnia/grpc#use-buf-schema-registry-reflection

Also, here's the blog post that dives into more of the technical details:
https://buf.build/blog/kong-insomnia-grpc-bsr-support

**Comment by mirzakhany** [comment]
Hi @perezd 👋 
Thanks for showing interest in this project. And library you mentioned looks interesting. I’ll look into in deeper soon.
